### ISSUE TITLE
`rdfs:range xsd:string, rdf:Resource, xsd:Integer`

### ISSUE THREAD

**Issue by akuckartz** [ColinMaudry/open-contracting-ld#2]
```
:former_value a rdf:Property ;
  rdfs:label "Former value"@en ;
  rdfs:comment "he previous value of the changed property, in whatever type the property is."@en ;
  rdfs:range xsd:string, rdf:Resource, xsd:Integer ;
  rdfs:domain :Change .
```

That range seems to be empty (intersection of disjoint sets).

**Comment by ColinMaudry** [comment]
You're right, I had a doubt, but indeed multiple values means intersection:

https://www.w3.org/TR/rdf-schema/#ch_range

> Where P has more than one rdfs:range property, then the resources denoted by the objects of triples with predicate P are instances of all the classes stated by the rdfs:range properties.

I'll consequently remove the range, implying that the value can be anything.

**Comment by akuckartz** [comment]
If the intention is that the range should include both strings and URLs then there is another issue: How do you know when "https://example.org/test" is a string and when it is a URL ?

**Comment by ColinMaudry** [comment]
I guess you mean: how do you know, when converting JSON OCDS data to RDF, that the value is a string or a URL or something else?

It wouldn't be perfect, but a string that starts with "http://" or "https://" and has no spaces in the middle is usually a URL.

**Comment by akuckartz** [comment]
The problem then is that providing a JSON-LD context is not enough to convert OCDS data to RDF.
